interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Save Reports
+author: Mary Sammal
+---
+# Save Reports
+Create a report's _backup copy_ to guarantee that you can revert the report to its original state further.
+
+## Use the Ribbon's Home Tab
+
+* Click **Save** and the **Save as** to save your report's copy.
+	
+	![eurd-win-ribbon-save-report-as](../../../images/eurd-win-ribbon-save-report-as.png)
+	
+	Specify a folder and file name in the invoked standard **Save** dialog.
+
+* Click **Save** or press CTRL+S to save the report's current layout in the application's current folder.
```

Comment:
current... current?

Reply:
Click **Save** or press CTRL+S to save the report's layout in the application's current folder.

Ok?

Reply:
Click Save or press CTRL+S to save the report's layout in the application's folder.